packages/backend/src/projectAdapters/dbtBaseProjectAdapter.ts

```diff
@@ -232,4 +238,31 @@ export class DbtBaseProjectAdapter implements ProjectAdapter {
             [[] as DbtModelNode[], [] as ExploreError[]],
         );
     }
+
+    static async getWarehouseFromCache(
```

Comment:
Should I move these methods into a different *Model ? @owlas

Reply:
In the `projectModel` and then we could pass the `projectModel` in here.

Reply:
could I just import `projectModel from models` and use it here ?

Reply:
Or in the constructor, which would cause an interface change but it's only an internal interface, typescript should find all occurrences.